compiler/compile/VirtualGuard.cpp

```diff
@@ -196,8 +196,6 @@ TR_VirtualGuard::createVftGuard
    aconstNode->setInlinedSiteIndex(calleeIndex);
    aconstNode->setByteCodeIndex(0);
 
-   comp->verifySymbolHasBeenValidated(static_cast<void *>(thisClass));
```

Comment:
It would be better to squash this change with the one in #2977 that introduced the function in the first place...

Reply:
Yeah you're right, I was trying to avoid merge conflicts :) . I'll try and clean this up.